docs/plugins/analysis-nori.asciidoc

```diff
@@ -434,3 +434,52 @@ Which responds with:
 --------------------------------------------------
 
 <1> The Hanja form is replaced by the Hangul translation.
+
+
+[[analysis-nori-number]]
+==== `nori_number` token filter
```

Comment:
We also need to expose `discard_punctuation` in the tokenizer in order to detect floating point numbers and to split numbers correctly if they are separated with whitespaces ? We should then advise in the documentation to set `discard_punctuation` to `false` when using the `nori_number` filter and to remove punctuations in a subsequent filter ?

Reply:
I think so. 
I added more description for that.
Please check the additional commits.